algorithms/unit_tests/TestStdAlgorithmsTeamNoneOf.cpp

```diff
@@ -0,0 +1,179 @@
+        m_noneOfResultsView(noneOfResultsView),
+        m_apiPick(apiPick),
+        m_unaryOp(std::move(unaryOp)) {}
+
+  template <class MemberType>
+  KOKKOS_INLINE_FUNCTION void operator()(const MemberType& member) const {
+    const auto myRowIndex = member.league_rank();
+
+    auto myRowViewFrom = Kokkos::subview(m_dataView, myRowIndex, Kokkos::ALL());
+
+    switch (m_apiPick) {
+      case 0: {
+        const bool result = KE::none_of(member, KE::cbegin(myRowViewFrom),
+                                        KE::cend(myRowViewFrom), m_unaryOp);
+        Kokkos::single(Kokkos::PerTeam(member),
+                       [=]() { m_noneOfResultsView(myRowIndex) = result; });
+        break;
+      }
+
+      case 1: {
+        const bool result = KE::none_of(member, myRowViewFrom, m_unaryOp);
+        Kokkos::single(Kokkos::PerTeam(member),
+                       [=]() { m_noneOfResultsView(myRowIndex) = result; });
+        break;
+        break;
+      }
+    }
+  }
+};
+
+template <class LayoutTag, class ValueType>
+void test_A(std::size_t numTeams, std::size_t numCols, int apiId) {
+  /* description:
+     use a rank-2 view randomly filled with values,
+     and run a team-level none_of
+   */
+
+  // -----------------------------------------------
+  // prepare data
+  // -----------------------------------------------
+  // create a view in the memory space associated with default exespace
+  // with as many rows as the number of teams and fill it with random
+  // values from an arbitrary range.
+  constexpr auto lowerBound = ValueType{5};
+  constexpr auto upperBound = ValueType{523};
+  Kokkos::pair bounds{lowerBound, upperBound};
+  auto [dataView, dataViewBeforeOp_h] = create_random_view_and_host_clone(
+      LayoutTag{}, numTeams, numCols, bounds, "dataView");
+
+  // -----------------------------------------------
+  // launch kokkos kernel
+  // -----------------------------------------------
+  using space_t = Kokkos::DefaultExecutionSpace;
+  Kokkos::TeamPolicy<space_t> policy(numTeams, Kokkos::AUTO());
+
+  // to verify that things work, each team stores the result of its none_of
+  // call, and then we check that these match what we expect
+  Kokkos::View<bool*> noneOfResultsView("noneOfResultsView", numTeams);
+
+  GreaterThanValueFunctor unaryPred{upperBound};
+
+  // use CTAD for functor
+  TestFunctorA fnc(dataView, noneOfResultsView, apiId, unaryPred);
+  Kokkos::parallel_for(policy, fnc);
+
+  // -----------------------------------------------
+  // run cpp-std kernel and check
+  // -----------------------------------------------
+  auto noneOfResultsView_h = create_host_space_copy(noneOfResultsView);
+
+  for (std::size_t i = 0; i < dataView.extent(0); ++i) {
+    auto rowFrom = Kokkos::subview(dataViewBeforeOp_h, i, Kokkos::ALL());
+    const bool result =
+        std::none_of(KE::begin(rowFrom), KE::end(rowFrom), unaryPred);
+    EXPECT_EQ(result, noneOfResultsView_h(i));
+    break;
```

Comment:
why you break here? if you want to use ASSERT_EQ that is fine too

Reply:
That's just a mistake. I'll remove it.

Reply:
Done.